util/tlgen/xbar.sim_cfg.hjson.tpl

```diff
@@ -7,6 +7,12 @@
   // Top level dut name (sv module).
   dut: xbar_${xbar.name}
 
+  // Testplan hjson file.
+  testplan: "{proj_root}/hw/${xbar.top_name}/ip/{dut}/data/autogen/{dut}_testplan.hjson"
+
+  // Add xbar specific exclusion files.
+  vcs_cov_excl_files: ["{proj_root}/hw/${xbar.top_name}/ip/{dut}/dv/cov/xbar_cov_excl.el"]
+
```

Comment:
If anything is related to the top, then I recommend it to be generated in topgen. Same to `top_name` field. The value is assigned in the topgen, but revised in the tlgen data structure.